fulp_modules/features/antagonists/bloodsuckers/code/antagonists/vassal_datum.dm

```diff
@@ -180,6 +199,19 @@
 	for(var/datum/action/cooldown/bloodsucker/power in powers)
 		power.level_current++
 
+/datum/antagonist/vassal/proc/show_hud()
+	var/datum/atom_hud/alternate_appearance/basic/has_antagonist/hud
+	for(var/datum/atom_hud/alternate_appearance/basic/has_antagonist/antag_hud as anything in GLOB.has_antagonist_huds)
+		if(antag_hud.target != owner.current)
+			continue
```

Comment:
can we check ``team_hud_ref`` here instead? you gotta just resolve it.

Reply:
> can we check `team_hud_ref` here instead? you gotta just resolve it.

how did i mis that